mabl snippets/isElementPresent.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * This snippet will allow you to assert against an element 
+ * as long as the display of the element changes from 'none' 
+ * to anything else.
+ * @param {object} - mablInputs Object containing input
+ *                   variables (mablInputs.variables.user)
+ * @param {function} callback - The callback function
+ */
+function mablJavaScriptStep(mablInputs, callback) {
+
+    // this is where you would define what your element is
+    let element = document.querySelector('REPLACE_WITH_CSS_SELECTOR');
+    
+    // conditional statement to assure that the element is present, as well as
+    // assuring the display does not equal 'none'
+    if (element && element.style.display !== 'none') {
```

Comment:
`element.style` only describes the style explicitly specified for the element (not anything applied via CSS), so a more general way is to use the computed style that is after all CSS has been applied.  and there are a couple other common things that result in non-displayed elements that you could include with something like:
```
  ...
  if (element) {
    const style = window.getComputedStyle(element);
    if (style && style.display !== 'none' && style.visibility !== 'collapse' && style.visibility !== 'hidden') {
      callback('Element Present');
    } else {
      throw Error('Element Not Visible');
    }
  } else {
    throw Error('Element Not Present');
  }
```